src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -656,7 +664,7 @@ def evaluate(self, dataset, metric = 'auto', verbose = True):
         
         return ret
 
-    def predict(self, data, verbose = True):
+    def predict(self, data, output_type='class', verbose=True):
```

Comment:
What does this have to do with improved error handing for validation sets? Please provide focuses PRs that only include the commits that need review. Otherwise, it's too easy for changes to get lost in the mix of what changes are duplicated and should be ignored. It invites sloppier and less accurate reviews.

Reply:
Sorry about that, rogue commit made its way into this branch. Deleted it now.